lib/css/components/_stacks-topbar.less

```diff
@@ -0,0 +1,366 @@
+
+    // Search switcher
+    --theme-topbar-select-color:               @black-700;
+    --theme-topbar-select-background:          @black-050;
+
+    // topbar items
+    --theme-topbar-item-color:                 @black-600;
+    --theme-topbar-item-color-hover:           @black-800;
+    --theme-topbar-item-background-hover:      @black-075;
+    --theme-topbar-item-color-current:         @black;
+    --theme-topbar-item-border-current:        var(--theme-primary-color);
+}
+
+// forced dark topbar
+.s-topbar__dark {
+    @topbar-actual-background: @black-800;
+    // TODO
+    #calc-topbar-lightness-increase() when (lightness(@topbar-actual-background) < 15%) {
+        @topbar-search-lightness-increase: 20%;
+    }
+    #calc-topbar-lightness-increase() when not (lightness(@topbar-actual-background) < 15%) {
+        @topbar-search-lightness-increase: 10%;
+    }
+    #calc-topbar-lightness-increase();
+
+    --theme-topbar-background-color: @topbar-actual-background;
+    // Search input
+    --theme-topbar-search-color:               lighten(@topbar-actual-background, 80% + @topbar-search-lightness-increase);
+    --theme-topbar-search-background:          lighten(@topbar-actual-background, @topbar-search-lightness-increase);
+    --theme-topbar-search-placeholder:         lighten(@topbar-actual-background, 60% + @topbar-search-lightness-increase);
+    --theme-topbar-search-border:              lighten(@topbar-actual-background, @topbar-search-lightness-increase);
+    --theme-topbar-search-border-focus:        lighten(@topbar-actual-background, 45% + @topbar-search-lightness-increase);
+    --theme-topbar-search-shadow-focus:        0 0 0 @su4 fade(@white, 30%);
+
+    // Search switcher
+    --theme-topbar-select-color:               lighten(@topbar-actual-background, 60% + @topbar-search-lightness-increase);
+    --theme-topbar-select-background:          lighten(@topbar-actual-background, 10% + @topbar-search-lightness-increase);
+
+    // topbar items
+    --theme-topbar-item-color:                 @black-025;
+    --theme-topbar-item-color-hover:           @white;
+    --theme-topbar-item-background-hover:      @black-700;
+    --theme-topbar-item-color-current:         @white;
+    --theme-topbar-item-border-current:        var(--theme-primary-color);
+
+    // themed border accent
+    --theme-topbar-accent-border:              none;
+}
+
+.s-topbar {
+    min-width: auto;
+    box-shadow: var(--bs-sm);
+    width: 100%;
+    z-index: @zi-navigation-fixed;
+    background-color: var(--theme-topbar-background-color);
+    height: var(--topbar-height);
+    display: flex;
+    border-top: var(--theme-topbar-accent-border);
+    position: relative;
+
+    // TODO necessary? used to wrap content so the topbar stretches 100% w/ content at some value below that
+    .s-topbar--container {
+        width: 100%; // TODO note - consumers should use atomic classes to override this
```

Comment:
Set it to `width: @s-full; max-width: 100%;`. This looks pretty good in the general case as well as on our squished examples in the docs. I'd prefer not to use `@site-width` since I'm actively trying to remove that and doing so would require me to move this component out of `static` and into `dynamic`, which kills my ability to use it in non-unified areas (SE.com)